src/content/Kentico.Kontent.Boilerplate/Areas/WebHooks/Controllers/WebhooksController.cs

```diff
@@ -46,7 +46,7 @@ public IActionResult Index([FromBody] WebhookModel model)
                     dependencies.Add(CacheHelper.GetTypesDependencyKey());
                 }
 
-                if (model.Message.Type == "content_type")
+                if (model.Message.Type.Equals("content_type", StringComparison.Ordinal))
```

Comment:
why equals?

Reply:
Yeah, this change doesn't make any sense as it's equivalent.

Reply:
i'd keep it simple then :D